app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -216,14 +216,13 @@ def build(toolbar)
 
       @sep_added = false
       @groups_added.push(group_index)
-
       case group
       when ApplicationHelper::Toolbar::Group
         group.buttons.each do |bgi|
           build_button(bgi, group_index)
         end
       when ApplicationHelper::Toolbar::Custom
-        group.render(@view_context)
+        group[:args][:html] = CGI.escapeHTML(group.render(@view_context)).delete!("\n")
```

Comment:
Ah okay, didn't knew that. Googled and found this sollution. But I'll use the `h`.